### DISCUSSION TITLE
How can I implement better save times and event handlers?

### DISCUSSION THREAD

**Discussion by hvaandres** [cypress-io/cypress#16260] [Questions and Help]
I'm currently running some tests within salesforce and cypress, and it seems that I'm hitting an issue inside of salesforce called "We hit a snag - Review the following fields - First Name" but the first name has been filled already. This only happens as soon as I run the same test twice.

![wehitasnag](https://user-images.githubusercontent.com/26829975/116484911-48a74c00-a847-11eb-9774-211919ca85a9.JPG)

I would like to know if someone has any idea of what you can do to skip this type of issue inside of salesforce.

**Comment by tit** [comment]
How do you fill in the field **First Name**?
